src/charm.py

```diff
@@ -52,9 +52,9 @@ class _ReplicaRelationNotReady(Exception):
         'mscom',
         'thematic',
         'twentyeleven',
-        'twentynineteen',
         'twentytwenty',
         'twentytwentyone',
+        'twentytwentytwo',
```

Comment:
Why this change?

Reply:
I bumped the WordPress version to 6.0.2, the default theme has changed so I need to update that in charm.